afs-ext-base/src/main/java/com/powsybl/afs/ext/base/ImportedCaseBuilder.java

```diff
@@ -102,6 +109,11 @@ public ImportedCaseBuilder withDatasource(ReadOnlyDataSource dataSource) {
         return this;
     }
 
+    public ImportedCaseBuilder withDatastore(ReadOnlyDataStore dataStore) {
```

Comment:
I think we should instead provide directly a `DataPack` here, because what we need is a consistent set of files.

Possibly we can also have an additional method to take directly a `ReadOnlyDataStore`, but then it would need a name to search for in the data store.